packages/nimble-components/src/select/template.ts

```diff
@@ -151,11 +153,19 @@ SelectOptions
                             })}
                         ></slot>
                     </div>
-                    ${when(x => (x.filterMode !== FilterMode.none && x.filteredOptions.length === 0), html<Select>`
+                    ${when(x => (x.filterMode !== FilterMode.none && x.filteredOptions.length === 0 && !x.loadingVisible), html<Select>`
                         <span class="no-results-label ${x => x.positionAttribute}">
                             ${x => filterNoResultsLabel.getValueFor(x)}
                         </span>
                     `)}
+                    ${when(x => x.loadingVisible, html<Select>`
+                        <div class="loading-container" @click="${(x, c) => x.ignoreClickHandler(c.event as MouseEvent)}">
```

Comment:
Do we need `ignoreClickHandler` here?

Reply:
It's here so the dropdown isn't dismissed if a user clicks that visual. This avoids closing on it on misclicks.